packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -173,6 +191,14 @@ export default {
         },
         showAriaLabel () {
             return this.ariaLabel === '' ? this.uid : this.ariaLabel;
+        },
+        hasChevronIcon () {
+            return this.closeButtonStyle === 'chevron';
+        },
```

Comment:
I would move the logic out of the view for this property and make it:

```suggestion
        hasBackButton () {
            return (this.closeButtonStyle === 'chevron' && this.hasCloseButton);
        },
```